lms/djangoapps/instructor_analytics/tests/test_basic.py

```diff
@@ -26,7 +26,8 @@
     sale_record_features, sale_order_record_features, enrolled_students_features,
     course_registration_features, coupon_codes_features, list_may_enroll,
     AVAILABLE_FEATURES, STUDENT_FEATURES, PROFILE_FEATURES,
-    get_proctored_exam_results)
+    get_proctored_exam_results,
```

Comment:
We actually don't need this import, the following line `from instructor_analytics.basic import student_responses` should do the trick for David's test